app/app.py

```diff
@@ -168,7 +168,7 @@ def get_where_clause(flow_key_init, flow_key_end, intersect_id_string, **params)
     id_filter = f"{flow_key_init} IN ({intersect_id_string}) AND {flow_key_init} NOT IN ('OUT_OF_BOUNDS') AND {flow_key_end} NOT IN ('OUT_OF_BOUNDS')"
 
     # exclude trips that don't meet our minimum criteria to be considered valid
-    trip_filter = " trip_distance * 0.000621371 >= 0.1 AND trip_distance * 0.000621371 < 500 AND trip_duration < 86400"
+    trip_filter = " AND trip_distance * 0.000621371 >= 0.1 AND trip_distance * 0.000621371 < 500 AND trip_duration < 86400"
```

Comment:
![and](https://user-images.githubusercontent.com/5697474/50911482-976e1500-13f5-11e9-92a2-122584553317.gif)